core/modules/qdisp/Executive.cc

```diff
@@ -238,27 +199,27 @@ bool Executive::join() {
 }
 
 void Executive::markCompleted(int jobId, bool success) {
-    ResponseRequester::Error err;
+    ResponseHandler::Error err;
     LOGF(getLogger(), LOG_LVL_INFO, "Executive::markCompleted(%1%,%2%)" % jobId % success);
     if(!success) {
         {
-            std::lock_guard<std::mutex> lock(_requestersMutex);
-            RequesterMap::iterator i = _requesters.find(jobId);
-            if(i != _requesters.end()) {
-                err = i->second->getError();
+            std::lock_guard<std::mutex> lock(_incompleteJobsMutex);
+            auto i = _incompleteJobs.find(jobId);
```

Comment:
Compiler can sure tell what is the type of `i` here but I find it harder to understand after this change. Renaming variable to `iter` can help me to guess what to expect.